terraform/modules/forest_node/variable.tf

```diff
@@ -70,3 +70,20 @@ variable "fw_name" {
   description = "The name assigned to the firewall in the cloud"
   type        = string
 }
+
+variable "NR_LICENSE_KEY" {
+  description = "New Relic Access Token"
+  type        = string
+}
+
+variable "NEW_RELIC_API_KEY" {
+  description = "New Relic API KEY"
+  type        = string
+}
+
+variable "NEW_RELIC_ACCOUNT_ID" {
+  description = "New Relic Account ID"
+  type        = string
+  default     = "3942575"
```

Comment:
Is it our account id?

Reply:
Why do we expose our account id?

Reply:
1. The account may change for any reason. The ID should be easily changeable.
2. I don't know NR's policy on hiding the details of accounts (and no policies can protect accounts from leaks). It's best to not share even such things as an ID for privacy reasons.